decidim-elections/lib/decidim/elections/component.rb

```diff
@@ -335,7 +335,20 @@
 
           Decidim::Elections::Result.create!(
             votes_count: Faker::Number.number(digits: 3),
-            answer: answer
+            election: election_with_results,
+            question: result_question,
+            answer: answer,
+            result_type: "valid_answer"
+          )
+        end
+
+        if Faker::Boolean.boolean(true_ratio: 0.2)
+          Decidim::Elections::Result.create!(
+            votes_count: Faker::Number.number(digits: 3),
+            election: election_with_results,
+            question: result_question,
+            answer: nil,
+            result_type: "blank_question"
```

Comment:
💬 I'm not sure, but maybe `blank_answer` would be more accurate? 🤔

Reply:
I agree, I can change the naming. But at first I thought it was confusing to name it `blank_answer` with no relation to an answer.